commands/publish_dafsa.py

```diff
@@ -0,0 +1,106 @@
+import os
+import json
+import tempfile
+import subprocess
+import requests
+from requests.exceptions import HTTPError
+
+from kinto_http import Client, KintoException
+
+
+PSL_FILENAME = "public_suffix_list.dat"
+
+COMMIT_HASH_URL = (
+    f"https://api.github.com/repos/publicsuffix/list/commits?path={PSL_FILENAME}"
+)
+LIST_URL = f"https://raw.githubusercontent.com/publicsuffix/list/master/{PSL_FILENAME}"
+
+MAKE_DAFSA_PY = "https://raw.githubusercontent.com/arpit73/temp_dafsa_testing_repo/master/publishing/make_dafsa.py"
+PREPARE_TLDS_PY = "https://raw.githubusercontent.com/arpit73/temp_dafsa_testing_repo/master/publishing/prepare_tlds.py"
+
+BUCKET_ID = "firefox-core-network-dns"
+COLLECTION_ID = "public-suffix-list"
+RECORD_ID = "latest-commit-hash"
+
+
+def get_latest_hash():
+    response = requests.get(COMMIT_HASH_URL)
+    response.raise_for_status()
+    return response.json()[0]["sha"]
+
+
+def download_resources(directory, *urls):
+    for url in urls:
+        # file_location is found by appending the file_name(at the end of url string) to temp directory
+        file_name = os.path.basename(url)
+        file_location = os.path.join(directory, file_name)
+        response = requests.get(url, stream=True)
+        response.raise_for_status()
+
+        with open(file_location, "wb") as f:
+            for chunk in response.iter_content(chunk_size=1024):
+                try:
+                    f.write(chunk)
+                except IOError as e:
+                    raise Exception(f"IO Error Occurred: {e}")
```

Comment:
I don't understand this. Why wouldn't we want to let the `IOError` be raised? What is the advantage of loosing all the information of the IOError and get a generic exception with only a message?